packages/test/src/test-native-connection.ts

```diff
@@ -21,7 +21,7 @@ if (RUN_INTEGRATION_TESTS) {
   test('NativeConnection errors have detail', async (t) => {
     await t.throwsAsync(() => NativeConnection.connect({ address: 'localhost:1' }), {
       instanceOf: TransportError,
-      message: /.*Connection refused.*/,
+      message: /.*Connection[ ]?refused.*/i,
```

Comment:
When would the space be optional? Do we see the string `ConnectionRefused`?

Reply:
If I'm not mistaken, that one was on Windows.

Really, all of those fixes are things I discovered while working on the CI refactor branch. I extracted these to a distinct PR only so that git history remains sane. You know... "Why did we made that space optionnal? ... Because of a CI refactor?!? ehhhhh...." :smiley: